data-access/src/main/java/am/smartcafe/data_access/dto/resp/UserRegisterResponse.java

```diff
@@ -0,0 +1,88 @@
+package am.smartcafe.data_access.dto.resp;
+
+import am.smartcafe.data_access.model.Role;
+import lombok.Builder;
+
+import javax.validation.constraints.*;
+import java.util.Objects;
+import java.util.Set;
+
+@Builder
+public class UserRegisterResponse {
```

Comment:
do we need validation on response?

Reply:
UserRegisterResponse used in the UserService

Reply:
as I understand we send this response back after successfully registration right?

Reply:
yes, that's how I wrote the logic

Reply:
Ok, but I think we don't need to send back validated information if it validated already, is it?

Reply:
Yes, you are right, I may have made a mistake in that part

Reply:
it's ok, may happen